packages/thumbprint-react/components/Alert/index.module.scss

```diff
@@ -0,0 +1,66 @@
+@import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
+
+.tpAlert {
+    display: flex;
+    font-size: $tp-font__body__2__size;
+    line-height: $tp-font__body__2__line-height;
+    text-align: left;
+    padding: 12px $tp-space__3;
+    color: $tp-color__white;
+    background: $tp-color__gray;
+    font-weight: 700;
+    border-radius: $tp-border-radius__base;
+
+    & a {
+        text-decoration: underline;
+        cursor: pointer;
+        color: $tp-color__white;
+        fill: $tp-color__white;
+
+        &:hover {
+            color: $tp-color__white;
+            opacity: 0.7;
+        }
+    }
+
+    &.banner {
```

Comment:
With CSS modules, there's no need to use the `&` operator, since all class names are scoped to the file that imports them. You can flatten all of this out.